php/commands/src/CLI_Command.php

```diff
@@ -436,7 +467,35 @@ private function get_updates( $assoc_args ) {
 				continue;
 			}
 
-			if ( ! isset( $release->assets[0]->browser_download_url ) ) {
+			$package_url = null;
+
+			foreach ( $release->assets as $asset ) {
+				if ( ! isset( $asset->browser_download_url ) ) {
+					continue;
+				}
+
+				if ( substr( $asset->browser_download_url, - strlen( '.phar' ) ) === '.phar' ) {
+					$package_url = $asset->browser_download_url;
+				}
+
+				// The manifest.json file, if it exists, contains information about PHP version requirements and similar.
+				if ( substr( $asset->browser_download_url, - strlen( 'manifest.json' ) ) === 'manifest.json' ) {
+					$response = Utils\http_request( 'GET', $asset->browser_download_url, null, $headers, $options );
+
+					if ( $response->success ) {
+						$manifest_data = json_decode( $response->body );
+
+						if (
+							isset( $manifest_data->requires_php ) &&
+							! Comparator::greaterThanOrEqualTo( PHP_VERSION, $manifest_data->requires_php )
```

Comment:
Should this use built in [version_compare](https://www.php.net/manual/en/function.version-compare.php) (as we do elsewhere)?

Reply:
We already use Comparator elsewhere in this file. Not sure about the historical reason.